### DISCUSSION TITLE
How to interpret "<" in  Orig. start

### DISCUSSION THREAD

**Discussion by chasemc** [antismash/antismash#479] [Q&A] [answered]
Sorry if this is a dumb question but I'm not sure how to interpret "Orig. start" being "<" in GenBank output files. How would I map that back to the original sequence?
e.g.
>COMMENT     ##antiSMASH-Data-START##
            Version      :: 6.1.1
            Run date     :: 2022-10-22 10:01:11
            NOTE: This is a single cluster extracted from a larger record!
            **Orig. start  :: <1082390**
            Orig. end    :: 1092264
            ##antiSMASH-Data-END##

**Comment by SJShaw** [comment, marked as answer]
That will happen if the first CDS in the region has an ambiguous start position (i.e. that CDS will have a location of `<1082390..end` in your original input).

If you can refine the gene calling for that gene to have accurate coordinates, you'll have a matching accurate region start coordinate.

**Comment by chasemc** [comment]
Thanks, that make sense. It's to be written in a tool so no control over input.
